content/_index.md

```diff
@@ -2,29 +2,41 @@
 title: Home
 ---
 {{< home/section title="What?" iconclass="fas fa-2x fa-power-off">}}
-**GitBOM** is creative re-purposing of [git](https://en.wikipedia.org/wiki/Git)'s [directed acyclic graph](https://en.wikipedia.org/wiki/Directed_acyclic_graph) to enable **automatic verifiable artifact resolution** across today's language-heterogeneous open source supply chains.
+
+**[GitBOM](/glossary/gitbom)** is a minimalistic scheme for [build tools](/glossary/build_tool) to:
+1. Build a compact [artifact tree](/glossary/artifact_tree), tracking every source code file incorporated into each built [artifact](/glossary/artifact).
+2. Embed a unique, content-addressable reference for that [artifact tree](/glossary/artifact_tree/), the [GitBOM identifier](/glossary/gitbom/#gitbom-identifier), into the [artifact](/glossary/artifact) at build time.
+
+[GitBOM](/glossary/gitbom) is designed to:
+- Consistently construct verifiable [artifact trees](/glossary/artifact_tree) across languages, environments, and packaging formats, with zero developer effort, involvement, or awareness
+- Enable automatic, verifiable [artifact](/glossary/artifact) resolution across today's diverse software supply chains
+- Complement SBOMs, such as [SPDX](https://spdx.dev/), [CycloneDX](https://cyclonedx.org/), or [SWID](https://nvd.nist.gov/products/swid)
+- Co-exist with, but not require, version control systems
 
 {{% notification type="info" %}}
-**Note that GitBOM is not a replacement for [SBOM](/glossary/sbom)s**, nor is it a version control system or a signing scheme.
+**[GitBOM](/glossary/gitbom) is NOT** (contrary to the name's appearance):
+- [Git](/glossary/git)
+- An [SBOM](/resources/glossary/#sbom), nor a replacement for SBOMs
+- A version control system
+- A signing scheme
 
 It is compatible with, and augments, these classes of tools.
 {{% /notification %}}
 
-Contrary to the name's appearance, GitBOM is neither [git](/glossary/git) nor an [SBOM](/glossary/sbom) -- it is a minimalistic scheme for all [build tools](/glossary/build_tool) to:
+[GitBOM](/glossary/gitbom) applies the [Unix Philosophy](https://en.wikipedia.org/wiki/Unix_philosophy) of "do one thing, and do it well."
```

Comment:
```suggestion
```

Reply:
re-read this a few times and I keep coming back -- I feel like this line belongs in the "Why?" section

Reply:
Makes sense to me